taxcalc/calculate.py

```diff
@@ -275,6 +275,48 @@ def mtr(self, income_type_str='e00200p',
         # return the three marginal tax rate arrays
         return (mtr_fica, mtr_iit, mtr_combined)
 
+    def capital_mtr(self):
+        capital_income_sources = ['e00300', 'e00400', 'e00600', 'e00650',
```

Comment:
It might be easier to test for consistency against the other mtr function (until we merge the two) if this list were a default argument. Then you could have a couple of kinds of tests. 

1. pass `['e00300']` to this function and make sure you get the same as when you use the preexisting `mtr()`. 

2. pass `['e00300', 'e2350']` to this function and make sure that you get the same thing as if you calculate the mtrs separately for those variables via the preexisting `mtr()` and then weight the mtrs after the fact.